docs/reference/bidi.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BiDi Protocol Support
+---
+
+XCUITest driver has partial support of the [BiDi Protocol](https://w3c.github.io/webdriver-bidi/) since version 7.26.0.
+Only events and commands mentioned below are supported.
+All other entities described in the spec throw not implemented errors.
+
+# Supported Events
+
+## log.entryAdded
+
+This event is emitted if the driver retrieves a new entry for any of the below log types. Logs collection might be disabled by the `appium:skipLogCapture` capability.
+
+### crashlog
+
+Events are emitted for both emulator and real devices. The latter only works if [py-ios-device](hhttps://github.com/YueChen-C/py-ios-device) is installed on the server host. Each event contains a particular device crash report entry.
+Events are always emitted with the `NATIVE_APP` context.
```

Comment:
Not really an issue, but lines separated by a single newline will be shown on the same line in MkDocs. New paragraphs require two newlines. The current format might be a bit difficult to read with the sentences all starting with 'Events'.
I think using bulleted lists would make it cleaner.

Reply:
It's ok if it is in the same paragraph. I just prefer to make .md files in such style